### ISSUE TITLE
Documentaion with Ideation

### ISSUE THREAD

**Issue by ghost** [theupweb/idea-lab#6]
Documentation with Team Requirements and exploring all the possible ways to make this project in a better way.

Check #3 for Ideas and Add in the Documentation

**Comment by ankita413** [comment]
Okay

**Comment by nitya123-github** [comment]
yeah okay!

**Comment by ankita413** [comment]
guys if we have any doubts how do we communicate, like is there any group or something

**Comment by ghost** [comment]
Hey @ankita413 , Join this conversation. https://discord.com/channels/731001496417730611/790973908366000199/792453736051507220

**Comment by ghost** [comment]
@JoshuaPoddoku link is just discord site.
Cam you send invitation link?

**Comment by ghost** [comment]
@ankita413 
Yeah you need to login to discord and here is the invite link: https://discord.gg/eVYHNse

**Check** #8

**Comment by ankita413** [comment]
Sorry but i am getting invite is invalid

**Comment by chehak123** [comment]
hii 
i would like to work on this issue

**Comment by ghost** [comment]
> @JoshuaPoddoku link is just discord site.
> Cam you send the invitation link?

https://discord.com/invite/3fRv4qVanA 

@ankita413 check this

**Comment by ghost** [comment]
@chehak123 Please check the readme file and you can start a discussion in Discord: https://discord.com/invite/3fRv4qVanA

**Comment by kanishkaa24** [comment]
I want to work on this issue.

**Comment by abhinav-huria** [comment]
I would like to work on this.

**Comment by Dhriti-Roy** [comment]
I would like to work on this issue
